src/platforms/common/configuration/options.mdx

```diff
@@ -182,6 +182,17 @@ When set to `true`, the SDK will send session events to Sentry. This is supporte
 
 </ConfigKey>
 
+<ConfigKey name="consider-frames-in-app-by-default" supported={["dart"]}>
```

Comment:
if we agree on renaming, `considerInAppFramesByDefault`, we need to fix key names and param names here

Reply:
```suggestion
<ConfigKey name="consider-in-app-frames-by-default" supported={["dart"]}>
```